modules/geo-layers/src/index.js

```diff
@@ -27,4 +27,7 @@ export {default as H3ClusterLayer} from './h3-layers/h3-cluster-layer';
 export {default as H3HexagonLayer} from './h3-layers/h3-hexagon-layer';
 export {default as Tile3DLayer} from './tile-3d-layer/tile-3d-layer';
 export {default as TerrainLayer} from './terrain-layer/terrain-layer';
-export {default as MVTLayer} from './mvt-layer/mvt-layer';
+export {
```

Comment:
Don't export things just for testing purpose. You can import directly from `@deck.gl/geo-layers/mvt-layer/mvt-layer` in a spec file.